get-config.js

```diff
@@ -39,15 +40,14 @@ const getConfig = (config, provider, servicePath) => {
   const functionConfigurations = config.functionConfigurations || [];
   const mappingTemplates = config.mappingTemplates || [];
 
-  const schemaPath = path.join(
-    servicePath,
-    config.schema || 'schema.graphql',
-  );
+  const readSchemaFile =
+      (schemaRelPath) => fs.readFileSync(path.join(servicePath, schemaRelPath), {encoding: 'utf8'});
+
+  const schemaContent =
+    Array.isArray(config.schema) ?
+      mergeTypes(config.schema.map(readSchemaFile)) :
+      readSchemaFile(config.schema || 'schema.graphql');
```

Comment:
@trilliput 

> There are two ways you can use this package:
> 
> manually import each type
> import everything from a specified folder

It would be great if we could also support importing files from a folder.

Reply:
@bboure I'm not sure I'll have time this week, even though it's not complex at all. Probably the next week.

Reply:
thanks @trilliput